modules/data_team_helper/jsx/behavioural_qc_module.js

```diff
@@ -143,6 +143,32 @@ var IncompleteCandidatesRow = React.createClass({
 });
 
 var InstrumentConflictsRow = React.createClass({
+    handleClick: function handleClick(event) {
```

Comment:
This appears to be needless duplication. Can this use a closure returned from `loris.loadFilteredMenuClickHandler` instead?